my-app/src/app/main-layout/content-projection/course/course.component.spec.ts

```diff
@@ -16,10 +17,31 @@ describe('CourseComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(CourseComponent);
     component = fixture.componentInstance;
+    component.course = new Course(1, 'Course title', new Date(), 100, 'Course description');
```

Comment:
I would advice using object declaration 
component.course = { id: 1, title: "Course title" ...
First of all you may not have a possibility to use a constructor. 
Most likely the course will be created on BE. And you will have just a model for mocks.
The second point is you need model changing affect unit tests. If someone changes common model and only a part of the app, other parts' unit tests should fail because of type missmatch first of all.
So it should be a trigger, that changing a global model affected this component

Reply:
sure. thanks